src/data-table/types.ts

```diff
@@ -9,13 +9,18 @@ export interface DataTableProps {
     /** css class to put on table **/
     className?: string | undefined;
     selectedIds?: string[] | undefined;
+    disabledIds?: string[] | undefined;
     /** enable checkboxes */
     enableMultiSelect?: boolean | undefined;
     /** display select all checkbox */
     enableSelectAll?: boolean | undefined;
+    /** display action bar */
+    enableSelectionBar?: boolean | undefined;
+    selectionBarContent?: (() => ReactNode) | undefined;
```

Comment:
the standard JSX.Element should be sufficient in this case

```suggestion
    selectionBarContent?: ReactNode | undefined;
```